src/steps/10-FinancialDetails/FundingPlan.vue

```diff
@@ -1,14 +1,21 @@
 
 <template>
   <div>
-    Future Funding Plan page
+    <ATATFileUpload 
+    :validFileFormats="['xlsx', 'xls','pdf']"
+    id="FundingPlan"/>
```

Comment:
indent lines 5 and 6, drop closing `/>` to be aligned with opening `<`tag

Reply:
- [x] Done